lib/Marlin/Marlin/src/module/configuration_store.cpp

```diff
@@ -126,7 +132,25 @@ typedef struct {     bool X, Y, Z, X2, Y2, Z2, Z3, E0, E1, E2, E3, E4, E5; } tmc
 
 // Defaults for reset / fill in on load
 static const uint32_t   _DMA[] PROGMEM = DEFAULT_MAX_ACCELERATION;
-static const float     _DASU[] PROGMEM = DEFAULT_AXIS_STEPS_PER_UNIT;
+#if ENABLED(USE_PRUSA_EEPROM_AS_SOURCE_OF_DEFAULT_VALUES) 
+static float get_steps_per_unit(size_t index) {
```

Comment:
are you sure this is not called at runtime in some time-critical function (like stepper or planner)?

Reply:
yes
it is called only in
`void MarlinSettings::reset()`